Tests/MusicXMLTests/ScoreTests/README.md

```diff
@@ -0,0 +1,28 @@
+# ScoreTests
+
+## Test Case Structure
+The score tests consist of test suite executor and test cases. Any individual test case is a folder that satisfy the following rules.
+
+- The folder name is arbitrary
+- The folder must contain a `manifest.plist` file, otherwise the folder is not treated as a test case and thus skipped.
+
+### Manifest
+The `manifest.plist` further defines the test case.
+- The manifest is a dictionary at its root.
+- The manifest MUST contain a `DisplayName` String field to specify the test name.
+- The manifest MUST contain an `XMLFileName` String field to specify the test file name. The file name includes the file name and the extension. An uncompressed score file with that name MUST exists in the same folder as the manifest.
+- The manifest MAY contain an `ExpectationClassName` String field to specify the Swift class name that contains the expectation of the parsed MusicXML. The expectation class is elaborated in the next section. If not provided, equal check will not be conducted.
+- The manifest MAY contain an `Description` String field for a textual description of the score.
+- The manifest MAY contain an `Source` String field for an URL link to the source.
+
+
+### Expectation Class
+- The expectation MUST be a `class`. This is a restriction since `struct` is not dynamically lodable.
+- The expectation class MAY be arbitrarily named, but is recommended to follow `<TestCaseName>ParsingExpectation` naming scheme. This class name must be the same as `ExpectationClassName` contained in the manifest.
+- The expectation class MUST conform to `ScoreTestExpectation` protocol and provide a `MusicXML` object as the expected result of parsing.
+
+## ScoreTestSuite
+
+The test suite runs a single `XCTestCase`, goes over all the test cases in the folder and test parsing and equality for each one. It doesn't `XCTFail()` on failure, but instead prints error message in the console.
```

Comment:
As mentioned elsewhere, I think it would be nice to make `XCTFail()`ing optional via a configuration point in the `manifest` file.

Reply:
I would change this: 

> The test suite runs a single `XCTestCase`, goes over all the test cases in the folder and test parsing and equality for each one.

to:

> The test suite runs a single `XCTestCase` which tests parsing and equality for each test sample in the folder.

Reply:
> As mentioned elsewhere, I think it would be nice to make `XCTFail()`ing optional via a configuration point in the `manifest` file.

Good idea. Will do